diff --git a/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeF.prefab b/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeF.prefab
index b4ae222..8522860 100644
--- a/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeF.prefab
+++ b/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeF.prefab
@@ -115,7 +115,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
 --- !u!1 &1209435626411526
 GameObject:
   m_ObjectHideFlags: 1
@@ -250,7 +250,11 @@ Transform:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1209435626411526}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+<<<<<<< HEAD
   m_LocalPosition: {x: 0, y: 1.3, z: 0}
+=======
+  m_LocalPosition: {x: 0, y: 1.1, z: 0}
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children: []
   m_Father: {fileID: 4730890884460648}
@@ -290,7 +294,11 @@ Transform:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1528989546858558}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+<<<<<<< HEAD
   m_LocalPosition: {x: 0, y: 1.6, z: 0}
+=======
+  m_LocalPosition: {x: 0, y: 1.4, z: 0}
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children: []
   m_Father: {fileID: 4730890884460648}
@@ -316,7 +324,7 @@ Transform:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1075749433478672}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
-  m_LocalPosition: {x: 0, y: 1.4, z: 0}
+  m_LocalPosition: {x: 0, y: 1.2, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children: []
   m_Father: {fileID: 4730890884460648}
@@ -378,7 +386,7 @@ Transform:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1920536170188698}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
-  m_LocalPosition: {x: 0, y: 1.4, z: 0}
+  m_LocalPosition: {x: 0, y: 1.2, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children: []
   m_Father: {fileID: 4730890884460648}
@@ -553,7 +561,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -418025584
+=======
+  randomSeed: 1149071133
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -705,7 +717,11 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 2
+=======
+      scalar: 2.2
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -746,7 +762,11 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 2
+=======
+      scalar: 2.2
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -787,7 +807,11 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 2
+=======
+      scalar: 2.2
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -3815,7 +3839,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -897901349
+=======
+  randomSeed: 2093819757
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -7080,7 +7108,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -1746633798
+=======
+  randomSeed: -413754528
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -10345,7 +10377,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: 1674988868
+=======
+  randomSeed: 1767326608
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -13610,7 +13646,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -530087624
+=======
+  randomSeed: 1333769332
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -13762,7 +13802,7 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.75
+      scalar: 2.2
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -13803,7 +13843,7 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.75
+      scalar: 2.2
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -13844,7 +13884,7 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.75
+      scalar: 2.2
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -16872,7 +16912,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -1486438962
+=======
+  randomSeed: -1387620868
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -20134,7 +20178,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -808665823
+=======
+  randomSeed: 2088597063
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -23548,7 +23596,7 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.75
+      scalar: 2.2
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -23589,7 +23637,7 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.75
+      scalar: 2.2
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -23630,7 +23678,7 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.75
+      scalar: 2.2
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -26658,7 +26706,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: 2132650709
+=======
+  randomSeed: 423548472
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
diff --git a/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeM.prefab b/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeM.prefab
index 757b742..7ace951 100644
--- a/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeM.prefab
+++ b/TDSBSG/Assets/_Scenes/Prefabs/Enemy_PatrollerOfficeM.prefab
@@ -94,7 +94,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
 --- !u!1 &1473759344721928
 GameObject:
   m_ObjectHideFlags: 0
@@ -165,7 +165,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
 --- !u!1 &1793386068269996
 GameObject:
   m_ObjectHideFlags: 1
@@ -216,7 +216,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
 --- !u!4 &4047559406177388
 Transform:
   m_ObjectHideFlags: 1
@@ -338,7 +338,11 @@ Transform:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1473759344721928}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+<<<<<<< HEAD
   m_LocalPosition: {x: 0, y: 0, z: 0}
+=======
+  m_LocalPosition: {x: 2, y: 0, z: 0}
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children:
   - {fileID: 4047559406177388}
@@ -705,7 +709,7 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.5
+      scalar: 1.7
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -746,7 +750,7 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.5
+      scalar: 1.7
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -787,7 +791,7 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.5
+      scalar: 1.7
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -3818,7 +3822,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -1539655953
+=======
+  randomSeed: 534332448
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -7080,7 +7088,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: 661349941
+=======
+  randomSeed: 585105483
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -10345,7 +10357,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -33033366
+=======
+  randomSeed: -972913455
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -10497,7 +10513,11 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.75
+=======
+      scalar: 1.8
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -10538,7 +10558,11 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.75
+=======
+      scalar: 1.8
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -10579,7 +10603,11 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.75
+=======
+      scalar: 1.8
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -13610,7 +13638,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: 1484504888
+=======
+  randomSeed: 1444551251
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -20286,7 +20318,11 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.75
+=======
+      scalar: 1.8
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -20327,7 +20363,11 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.75
+=======
+      scalar: 1.8
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -20368,7 +20408,11 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.75
+=======
+      scalar: 1.8
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -23399,7 +23443,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: 1657087390
+=======
+  randomSeed: 377343699
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -26813,7 +26861,11 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.5
+=======
+      scalar: 1.55
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -26854,7 +26906,11 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.5
+=======
+      scalar: 1.55
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -26895,7 +26951,11 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
+<<<<<<< HEAD
       scalar: 1.5
+=======
+      scalar: 1.55
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -27714,7 +27774,11 @@ ParticleSystem:
     flipV: 0
     randomRow: 1
     sprites:
+<<<<<<< HEAD
     - sprite: {fileID: 21300006, guid: d6398b8d4fce1a34eb090afb2b4bff64, type: 3}
+=======
+    - sprite: {fileID: 21300002, guid: d6398b8d4fce1a34eb090afb2b4bff64, type: 3}
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   VelocityModule:
     enabled: 0
     x:
@@ -29923,7 +29987,11 @@ ParticleSystem:
   moveWithTransform: 0
   moveWithCustomTransform: {fileID: 0}
   scalingMode: 1
+<<<<<<< HEAD
   randomSeed: -482763995
+=======
+  randomSeed: 1784664975
+>>>>>>> 1bf19630c237fbf02e2556c592b45172a776d1af
   InitialModule:
     serializedVersion: 3
     enabled: 1
@@ -30075,7 +30143,7 @@ ParticleSystem:
     startSize:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.5
+      scalar: 1.7
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -30116,7 +30184,7 @@ ParticleSystem:
     startSizeY:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.5
+      scalar: 1.7
       minScalar: 1
       maxCurve:
         serializedVersion: 2
@@ -30157,7 +30225,7 @@ ParticleSystem:
     startSizeZ:
       serializedVersion: 2
       minMaxState: 0
-      scalar: 1.5
+      scalar: 1.7
       minScalar: 1
       maxCurve:
         serializedVersion: 2
